pkg/kubectl/cmd/create.go

```diff
@@ -52,15 +52,24 @@ var (
 		cat pod.json | kubectl create -f -
 
 		# Edit the data in docker-registry.yaml in JSON using the v1 API format then create the resource using the edited data.
-		kubectl create -f docker-registry.yaml --edit --output-version=v1 -o json`)
+		kubectl create -f docker-registry.yaml --edit --output-version=v1 -o json
+
+		# Create a service using the definition in example-service.yaml.
+		kubectl create -f example-service.yaml
+
+		# Create a replication controller using the definition in example-controller.yaml.
+		kubectl create -f example-controller.yaml
```

Comment:
Use Deployment instead of Replication Controller. 
Combine the above two examples into one: `kubectl create -f example-service.yaml -f example-deployment.yaml`